torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1370,7 +1370,7 @@ void XLATensor::log1p_(XLATensor& input) {
 }
 
 XLATensor XLATensor::logdet(const XLATensor& input) {
-  return input.CreateFrom(ir::ops::LogDet(input.GetIrValue()));
+  return input.CreateFrom(ir::MakeNode<ir::ops::LogDet>(input.GetIrValue()));
```

Comment:
Sorry, maybe my comment was not clear.
You do not have to create a logdet.cpp/h files.
You can place that op within ops.cpp/h as such op requires no metadata.

Reply:
I thought it would be better to place this in separate files as we have done for other linear algebra operations, which is why I made the minor change to include `MakeNode`

Reply:
Not the rule we follow.
If an op takes no metadata, it goes in ops.cpp/h
There are a couple of special cases (where we might need to know the real class), but this does not apply.
Ie:

https://github.com/pytorch/xla/blob/b255f1d1584cbc936712d92a13b74fddecff34fd/torch_xla/csrc/ops/ops.cpp#L279

Reply:
Thank you for pointing that out, and sorry about the inconvenience. I'll make the appropriate changes soon.

Reply:
@dlibenzi I have addressed your comment.